src/render/graph.rs

```diff
@@ -314,8 +320,27 @@ impl Graph {
                 .for_each(|edge| {
                     let output_node = nodes.get_mut(&edge.other_id).unwrap();
                     output_node.has_inputs_connected = true;
-                    let signal = &node.outputs[edge.self_index];
-                    output_node.inputs[edge.other_index].add(signal, channel_interpretation);
+
+                    let connection = &node.outputs[edge.self_index];
+
+                    if !connection.is_silent() {
```

Comment:
Again this optimization for checking silence is probably not worth is because it is also present in the actual `add` method

Reply:
I would argue that's the same problem as for the `AudioParam`, but I have no numbers this time :)